src/DSharpPlus.Examples.Commands.Basics/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using DSharpPlus.Commands;
+using DSharpPlus.Commands.Processors.SlashCommands;
+using DSharpPlus.Commands.Processors.TextCommands;
+using DSharpPlus.Commands.Processors.TextCommands.Parsing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace DSharpPlus.Examples.Commands.Basics
+{
+    public static class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            ConfigurationBuilder configurationBuilder = new();
+            configurationBuilder.AddEnvironmentVariables("");
```

Comment:
this is, at a glance, rather questionable. we should also probably encourage a config file to at least be considered

Reply:
Config files... For an example bot? The purpose of an example bot is to demonstrate how to do XYZ while being easy to setup and run.

Also I think this might throw, if it hasn't been tested